swap/swap.go

```diff
@@ -120,14 +120,15 @@ func swapRotatingFileHandler(logdir string) (log.Handler, error) {
 }
 
 // new - swap constructor without integrity check
-func new(stateStore state.Store, prvkey *ecdsa.PrivateKey, backend contract.Backend, params *Params) *Swap {
+func new(stateStore state.Store, prvkey *ecdsa.PrivateKey, backend contract.Backend, params *Params, backendNetworkID uint64) *Swap {
 	auditLog = newLogger(params.LogPath)
 	return &Swap{
 		store:            stateStore,
 		peers:            make(map[enode.ID]*Peer),
 		backend:          backend,
 		owner:            createOwner(prvkey),
 		params:           params,
+		backendNetworkID: backendNetworkID,
```

Comment:
This isn't used anywhere, why do we need to store it?

Reply:
I will delete it. It is reminiscent of how we were previously storing things in the state store. We might need to put it back later, but for now, we indeed don't use it.